# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/process_thread_policy.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/process_thread_policy.h
--- a/security/sandbox/chromium/sandbox/win/src/process_thread_policy.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/process_thread_policy.h	Wed Jun 12 11:10:48 2019 +0100
@@ -9,10 +9,9 @@
 
 #include <string>
 
-#include "sandbox/win/src/policy_low_level.h"
-
 #include "base/strings/string16.h"
 #include "sandbox/win/src/crosscall_server.h"
+#include "sandbox/win/src/policy_low_level.h"
 #include "sandbox/win/src/sandbox_policy.h"
 
 namespace sandbox {
@@ -72,9 +71,9 @@
   // 'current_dir' : The CWD with which to spawn the child process.
   static DWORD CreateProcessWAction(EvalResult eval_result,
                                     const ClientInfo& client_info,
-                                    const base::string16 &app_name,
-                                    const base::string16 &command_line,
-                                    const base::string16 &current_dir,
+                                    const base::string16& app_name,
+                                    const base::string16& command_line,
+                                    const base::string16& current_dir,
                                     PROCESS_INFORMATION* process_info);
 
   // Processes a 'CreateThread()' request from the target.
@@ -90,5 +89,4 @@
 
 }  // namespace sandbox
 
-
 #endif  // SANDBOX_SRC_PROCESS_THREAD_POLICY_H_